grid-proxy/internal/explorer/server.go

```diff
@@ -171,7 +169,7 @@ func (a *App) getNodes(r *http.Request) (interface{}, mw.Response) {
 // @Param rented_by query int false "rented by twin id"
 // @Param available_for query int false "available for twin id"
 // @Param farm_ids query string false "List of farms separated by comma to fetch nodes from (e.g. '1,2,3')"
-// @Param certification_type query string false "certificate type NotCertified, Silver or Gold" Enums(NotCertified, Silver, Gold)
+// @Param certification_type query string false "certificate type" Enums(Certified, DIY)
```

Comment:
I think according to our [docs](https://manual.grid.tf/faq/faq.html#what-are-the-different-certifications-available-for-3node-servers-and-farms-what-are-the-gold-and-silver-certifications) it should be as it was, right?

Reply:
I couldn't verify this from the docs, is it applied for the farms only or for the farms and nodes? so I went to check the values in graphql scheme and it looks like we have `Gold` and `NotCertified` for farms and `Diy` and `Certified` for the nodes.
@xmonader  can we verifty this?

Reply:
@Mik-TF are you aware of any changes there?

Reply:
From what I know there aren't any changes. What @Omarabdul3ziz is saying with gold and notcertified for farms, and diy and certified for the nodes, should still be effective.